etl_framework/MySqlDatabase.py

```diff
@@ -1 +1,2 @@
-raise Exception("Class has moved. Import SqlDatabaseInterface from etl_framework.datastore_interfaces instead!!")
+raise Exception("Class has moved. Import `SqlDatabaseInterface` from \
+    `etl_framework.datastore_interfaces.sql_database_interface instead`!")
```

Comment:
@hdahme the location.  The exception is misleading because I copy pasted it from somewhere else and forgot to change it.  You want to import MySqlDatabase, not the inteface